### PR TITLE
Add tooltip to the instance composer buttons

### LINKED ISSUE #5478: [Composer] Add tooltips
Add Tooltips to the buttons in the Composer view 

Image

### PR DESCRIPTION
**Pull request by matborowczyk**
# Description

* Short description here *

closes #5478 

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [x] Attached issue to pull request
- [x] Changelog entry
- [x] Code is clear and sufficiently documented
- [x] Sufficient test cases (reproduces the bug/tests the requested feature)
- [x] Correct, in line with design
- [ ] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )

https://github.com/inmanta/web-console/assets/113331659/28f8d54f-3626-4411-bd54-edcd3fb4de93

### CODE REVIEW

File: src/UI/words.tsx [hunk 1]
```diff
@@ -157,6 +157,9 @@ const dict = {
   "inventory.form.placeholder.floatList": "1.1, 3.14, 4.3",
   "inventory.form.placeholder.stringList": "Add a list of values",
   "inventory.form.placeholder.dict": '{"key": "value"}',
+  "inventory.instanceComposer.labelButtonTooltip": "Toggle connection labels",
+  "inventory.instanceComposer.addInstanceButtonTooltip":
+    "Add entity into canvas",
```
Comment (LukasStordeur) [inline src/UI/words.tsx:162]: Add new service to the canvas

Comment (LukasStordeur) [reply src/UI/words.tsx:162]: But entity is used internally, it's not printed to the user as is.

Comment (LukasStordeur) [reply src/UI/words.tsx:162]: Instance could be a better alternative

File: src/UI/words.tsx [hunk 2]
```diff
@@ -157,6 +157,9 @@ const dict = {
   "inventory.form.placeholder.floatList": "1.1, 3.14, 4.3",
   "inventory.form.placeholder.stringList": "Add a list of values",
   "inventory.form.placeholder.dict": '{"key": "value"}',
+  "inventory.instanceComposer.labelButtonTooltip": "Toggle connection labels",
+  "inventory.instanceComposer.addInstanceButtonTooltip":
+    "Add Instance into canvas",
```
Comment (LukasStordeur) [inline src/UI/words.tsx:162]: Add new instance to the canvas.

Comment (matborowczyk) [reply src/UI/words.tsx]: resolved

**Review by LukasStordeur — APPROVED** [review]

**Pull request closed without merging**